### PR TITLE
Lesson about sprint board best practices

### PR DESCRIPTION
**Pull request by jmaddox19**
Perhaps I should be making this in Learn...? I guess this would be a good way for me to practice that. But I'm glad I got all my thoughts written out without that distraction. I'm happy to switch to that if that's what I ought to do.

### CODE REVIEW

**Comment by jmaddox19** [comment]
[[bEtsy] Write a resource about Trello best-practices](https://trello.com/c/zHnUE2JO/2227-betsy-write-a-resource-about-trello-best-practices)

**Comment by jmaddox19** [comment]
[Make skeleton of a Sprint board lesson](https://trello.com/c/qXZrVZUK/2230-make-skeleton-of-a-sprint-board-lesson)

File: developer-best-practices/using-sprint-boards.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Learning Goals
+- Understand what a sprint board is
+- Understand why sprint boards are often used in software development
+- Understand Ada's recommended best-practices for using sprint boards in Trello
+
+## What is a Sprint Board?
+TODO
+
+## Why is a Sprint Board?
+TODO
```
[the remark this answers is not in the document]
Comment (jmaddox19) [reply developer-best-practices/using-sprint-boards.md]: I like this idea. That could be a good precursor to the exercise. I will try to make a couple boards to link here before we push this (hopefully before this week is over).

File: 00-programming-fundamentals/using-planning-boards.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - This makes it easier to ensure every task you’re working on is directly related to the user story you’re currently focused on.
+    - This way, with each user story you complete, you know you already have this much to show off for your demo!
+1. Organize Trello cards by “Iteration/Week”. 
+    - This way you have some idea how you’re tracking: whether you’re ahead, behind, or on-track. 
+    - The easiest way to do this is probably to estimate what percentage of your project each user story represents and set a due date for it accordingly. 
+      - eg. If a story represents ~30% of what you have left to do for the project, it should be due in 13 days * .30 = 3.9 days (or around to 4 days).
+1. If there are one or two user stories that feel the scariest or riskiest, see if you can do those first. 
+    - That will give you more time to make mistakes and it will give you more time to change your idea if something seems like it isn’t quite going to work like you expected.
+
+  ## Exercise
```
Comment (CheezItMan) [inline 00-programming-fundamentals/using-planning-boards.md:27]: Maybe we give students an example Trello to examine and reference for these best-practices.

**Comment by jmaddox19** [comment]
The last thing I want to do for this PR to be complete is make an example Sprint board for bEtsy.
I think the Instructor should create one from scratch for the lesson but it will help to have an example to reference in preparation for the lesson.

**Comment by jmaddox19** [comment]
Ok @CheezItMan link added! This PR is finally ready for potential approval. If you could, I'd appreciate a feedback on the trello board itself as well. 
Thanks!

File: 00-programming-fundamentals/using-planning-boards.md (hunk 2 above)
Comment (dHelmgren) [reply 00-programming-fundamentals/using-planning-boards.md:28]: That, or pictures! Screenshots are cheap!

File: 00-programming-fundamentals/using-planning-boards.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    - This helps you as a developer focus on what matters to the user. Without articulating your work this way, it's SO EASY to get distracted by lots of technical issues that may or may not actually matter to the user your application is meant to serve.
+1. Sort user stories by priority and approach the project by focusing on completing one story at a time.
+    - For group projects, this means focusing on one story at a time per person.
+1. Use checklists on each user story to break the story up into smaller tasks. 
+    - This makes it easier to ensure every task you’re working on is directly related to the user story you’re currently focused on.
+    - This way, with each user story you complete, you know you already have this much to show off for your demo!
+1. Organize Trello cards by “Iteration", week, or priority. 
+    - Find some way of setting due dates for a piece of work
+      - eg. If a story represents ~30% of what you have left to do for the project and you have 13 more working days, it should be due in 13 days * .30 = 3.9 days (or around to 4 days).
```
Comment (dHelmgren) [inline 00-programming-fundamentals/using-planning-boards.md:22]: "Around to 4 days"
```suggestion
      - eg. If a story represents ~30% of what you have left to do for the project and you have 13 more working days, it should be due in 13 days * .30 = 3.9 days (or around 3 to 4 days).
```

File: 00-programming-fundamentals/using-planning-boards.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Using Planning Boards
+
+## Learning Goals
+- Understand Ada's recommended best-practices for using planning boards in Trello
+- Practice creating a planning board with tasks scheduled by iteration
+
+## Prerequisite
+This lesson will be hard to follow if students haven't first received the [Intro to Agile lesson](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/intro-to-agile.md)
+
+## Ada Best Practices
```
Comment (dHelmgren) [inline 00-programming-fundamentals/using-planning-boards.md:10]: Maybe we should call this something else? Maybe...
```suggestion
## Ada Suggested Practices
```

File: 00-programming-fundamentals/using-planning-boards.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+1. Create cards as User Stories 
+    - Format: "As a user, I can ...[, so that I...]". 
+        - eg. "As a user, I can browse products by merchant, so that I can explore all the cool stuff my favorite merchant sells and buy more stuff."
+    - This helps you as a developer focus on what matters to the user. Without articulating your work this way, it's SO EASY to get distracted by lots of technical issues that may or may not actually matter to the user your application is meant to serve.
```
Comment (dHelmgren) [inline 00-programming-fundamentals/using-planning-boards.md:14]:
```suggestion
    - This helps you as a developer focus on what matters to the user. Without articulating your work this way, it's SO EASY to get distracted by lots of technical issues that may or may not actually matter to the end user!
```

File: 00-programming-fundamentals/using-planning-boards.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+1. Sort user stories by priority and approach the project by focusing on completing one story at a time.
+    - For group projects, this means focusing on one story at a time per person.
+1. Use checklists on each user story to break the story up into smaller tasks. 
+    - This makes it easier to ensure every task you’re working on is directly related to the user story you’re currently focused on.
+    - This way, with each user story you complete, you know you already have this much to show off for your demo!
```
Comment (dHelmgren) [inline 00-programming-fundamentals/using-planning-boards.md:19]:
```suggestion
    - This way, with each user story you complete, you know you already have this much to show off for your demo!
    - If something in the checklist ends up being a big task in its own right, you can turn individual checklist items into their own cards!
```

File: 00-programming-fundamentals/using-planning-boards.md (hunk 2 above)
Comment (jmaddox19) [reply 00-programming-fundamentals/using-planning-boards.md:29]: The example board exists just a couple lines below.
